doc/Gabc.tex

```diff
@@ -490,17 +492,51 @@ \subsubsection{One-Note Neumes}
 {
   \gresetinitiallines{0}\gresetlyriccentering{firstletter}%
   \gresetclef{invisible}\gresetlastline{justified}\greseteolcustos{manual}%
+  \ttfamily
   \gabcsnippet{
     ( )
     g(g) g<sp>~~</sp>(g~) g<(g<) g>(g>) go(go) go<sp>~~</sp>(go~) gw(gw)
     gv(gv) gV(gV) gs(gs) gs<(gs<) g=(g=)
     ( ) (z) ( )
     gr(gr) gR(gR) gr0(gr0) G(G) G0(G0) G1(G1) G<sp>~~</sp>(G~) G>(G>) Gr(Gr) gx(gx)
-    g#(g#) gy(gy)
+    g#(g#) gy(gy) gx?(gx?) g#?(g#?) gy?(gy?)
     ( )
   }
 }
 
+\subsubsection{Alterations (Flats, Naturals, and Sharps)} \label{softalterations}
+
+Things like flats, sharps, and naturals (called ``alterations''), while not technically notes, are
+treated as one-note neumes by Gregorio\TeX.  Thus, they are also added by
+appending characters to pitch letters.
+
+Flats, sharps, and naturals each have a ``soft'' version (\verb|X|, \verb|##|, and \verb|Y|, respectively). Soft flats are printed if no previous flat (on the same pitch) is in effect. Similarly for soft sharps. Soft naturals are printed if a previous flat or sharp (on the same pitch) is in effect.
+What ``in effect'' means is controlled by the \verb|\gresetalterationeffect| macro (see \S\ref{setalterationeffect}). Soft alterations currently have two use cases.
+
+For Dominican chant (in which flats take effect until the end of a line or until cancelled by a natural), use \verb|\gresetalterationeffect{line}|, and type a soft flat (\verb|X|) in front of \emph{every} note that is to be sung a half-step flat. Naturals should also be soft (\verb|Y|), but only naturals that cancel flats need to be typed. For example:
+
+\begin{center}\parbox{5in}{%
+  \gresetinitiallines{0}\gresetlyriccentering{firstletter}%
+  \grechangestyle{translation}{\ttfamily}%
```

Comment:
There is no translation in the gabcsnippet. Does this line need to be here? Same with the `\grechangedim{translationheight}` two lines below.